source/MissionAction.cpp

```diff
@@ -186,11 +194,13 @@ void MissionAction::Load(const DataNode &node, const string &missionName)
 			conversation.Load(child);
 		else if(key == "conversation" && hasValue)
 			stockConversation = GameData::Conversations().Get(child.Token(1));
+		else if(key ==  "ship" && hasValue)
+			giftShips[GameData::Ships().Get(child.Token(1))] = (child.Size() > 2 ? child.Token(2) : "");
```

Comment:
`>=` is typically used over just `>` for checking the size of a DataNode.
```suggestion
			giftShips[GameData::Ships().Get(child.Token(1))] = child.Size() >= 3 ? child.Token(2) : "";
```

Reply:
Changed it in pursuit of a different problem, didn't know it was used more commonly this way.